Dart/src/com/jetbrains/lang/dart/ide/documentation/DartDocUtil.java

```diff
@@ -70,10 +70,11 @@ public static String generateDoc(final PsiElement element) {
   }
 
   private static String formatSignature(@NotNull final String signature) {
-    final int offsetToOpenParen = signature.indexOf('(');
+    // Match the first open paren, "(", but ignore a starting "(new)" or "(const)" patterns.
+    final int offsetToOpenParen = signature.startsWith("(") ? signature.indexOf('(', 2) : signature.indexOf('(');
```

Comment:
Looks like this could be as simple as `offsetToOpenParen = signature.indexOf('(', 1);`